includes/class-ajax.php

```diff
@@ -756,70 +753,64 @@ public function handle_frontend_submission() {
 
         $entry_fields = apply_filters( 'weforms_before_entry_submission', $entry_fields, $form, $form_settings, $form_fields );
 
-        $entry_id = weforms_insert_entry( [
-            'form_id' => $form_id,
-        ], $entry_fields );
-
+        $entry_id = 1;
+        $global_settings = weforms_get_settings();
+        if ( empty( $global_settings['after_submission'] ) ) {
+            $entry_id = weforms_insert_entry( [
+                'form_id' => $form_id,
+            ], $entry_fields );
         if ( is_wp_error( $entry_id ) ) {
             wp_send_json( [
                 'success' => false,
                 'error'   => $entry_id->get_error_message(),
             ] );
         }
-
-        // redirect URL
-        $show_message = false;
-        $redirect_to  = false;
-
-        if ( $form_settings['redirect_to'] == 'page' ) {
-            $redirect_to = get_permalink( $form_settings['page_id'] );
-        } elseif ( $form_settings['redirect_to'] == 'url' ) {
-            $redirect_to = $form_settings['url'];
-        } elseif ( $form_settings['redirect_to'] == 'same' ) {
-            $show_message = true;
-        } else {
-            $show_message = true;
-        }
-
-        // Fire a hook for integration
+        //Fire a hook for integration
         do_action( 'weforms_entry_submission', $entry_id, $form_id, $page_id, $form_settings );
-
-        $field_search = $field_replace = [];
-
-        foreach ( $form_fields as $r_field ) {
-            $field_search[] = '{' . $r_field['name'] . '}';
-
-            if ( $r_field['template'] == 'name_field' ) {
-                $field_replace[] = implode( ' ', explode( '|', $entry_fields[ $r_field['name'] ] ) );
-            } else if ( $r_field['template'] == 'address_field' ) {
-                $field_replace[] = implode( ', ', $entry_fields[ $r_field['name'] ] );
-            } else {
-                $field_replace[] = isset( $entry_fields[ $r_field['name'] ] ) ? $entry_fields[ $r_field['name'] ] : '';
-            }
-        }
-        $message = str_replace( $field_search, $field_replace, $form_settings['message'] );
-
-        // send the response
-        $response = apply_filters( 'weforms_entry_submission_response', [
-            'success'      => true,
-            'redirect_to'  => $redirect_to,
-            'show_message' => $show_message,
-            'message'      => $message,
-            'data'         => $_POST,
-            'form_id'      => $form_id,
-            'entry_id'     => $entry_id,
-        ] );
-
         $notification = new WeForms_Notification( [
-            'form_id'  => $form_id,
-            'page_id'  => $page_id,
+            'form_id' => $form_id,
+            'page_id' => $page_id,

```

Comment:
Why were the spaces removed?

Reply:
Not sure, added them back so that they line up